core/domain/platform_feature_services.py

```diff
@@ -71,6 +83,14 @@
             platform_parameter_list.ParamNames.
             HIGH_BOUNCE_RATE_TASK_MINIMUM_EXPLORATION_STARTS
         ),
+        (
+            platform_feature_list.ParamNames.
```

Comment:
Why are some of these "platform feature list" and others "platform parameter list"? What is the difference?

Some of the things labelled as "features" don't seem like boolean feature flags to me, so I guess that's not the distinction...

Reply:
There is a correction here, it should've been `platform_parameter_list` instead of `platform_feature_list`, I have corrected it.